airs/core/models/model.py

```diff
@@ -154,203 +201,205 @@ class DimensionType(Enum):
+    sar__frequency_band: str | None = Field(default=None, title="[STAC, extension sar] The common name for the frequency band to make it easier to search for bands across instruments. See section \"Common Frequency Band Names\" for a list of accepted names.")
+    sar__center_frequency: float | None = Field(default=None, title="[STAC, extension sar] The center frequency of the instrument, in gigahertz (GHz).")
+    sar__polarizations: str | None = Field(default=None, title="[STAC, extension sar] Any combination of polarizations.")
+    sar__product_type: str | None = Field(default=None, title="[STAC, extension sar] The product type, for example SSC, MGD, or SGC")
+    sar__resolution_range: float | None = Field(default=None, title="[STAC, extension sar] The range resolution, which is the maximum ability to distinguish two adjacent targets perpendicular to the flight path, in meters (m).")
+    sar__resolution_azimuth: float | None = Field(default=None, title="[STAC, extension sar] The azimuth resolution, which is the maximum ability to distinguish two adjacent targets parallel to the flight path, in meters (m).")
+    sar__pixel_spacing_range: float | None = Field(default=None, title="[STAC, extension sar] The range pixel spacing, which is the distance between adjacent pixels perpendicular to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type GRD.")
+    sar__pixel_spacing_azimuth: float | None = Field(default=None, title="[STAC, extension sar] The azimuth pixel spacing, which is the distance between adjacent pixels parallel to the flight path, in meters (m). Strongly RECOMMENDED to be specified for products of type GRD.")
+    sar__looks_range: float | None = Field(default=None, title="[STAC, extension sar] Number of range looks, which is the number of groups of signal samples (looks) perpendicular to the flight path.")
+    sar__looks_azimuth: float | None = Field(default=None, title="[STAC, extension sar] Number of azimuth looks, which is the number of groups of signal samples (looks) parallel to the flight path.")
+    sar__looks_equivalent_number: float | None = Field(default=None, title="[STAC, extension sar] The equivalent number of looks (ENL).")
+    sar__observation_direction: str | None = Field(default=None, title="[STAC, extension sar] Antenna pointing direction relative to the flight trajectory of the satellite, either left or right.")
+    proj__epsg: int | None = Field(default=None, title="[STAC, extension proj] EPSG code of the datasource.")
+    proj__wkt2: str | None = Field(default=None, title="[STAC, extension proj] PROJJSON object representing the Coordinate Reference System (CRS) that the proj:geometry and proj:bbox fields represent.")
```

Comment:
_proj__geometry_ and _proj__bbox_ ?

Reply:
`__` is used to encode `:`
The STAC field is `proj:geometry` but arlas (and python) does not support fields with `:`, so we user `__`